packages/client/src/generation/TSClient/Args.ts

```diff
@@ -144,10 +144,10 @@ export type ${modelArgName} = ${baseTypeName}
 
 export class MinimalArgsType implements Generatable {
   constructor(
-    protected readonly args: DMMF.SchemaArg[],
-    protected readonly type: DMMF.OutputType,
-    protected readonly action?: DMMF.ModelAction,
-    protected readonly collector?: ExportCollector,
```

Comment:
`ExportsCollector` is not used.

Reply:
Is it used elsewhere? We might be able to remove it completely. I did a search and it seems we append symbols to a collector in a few places `.addSymbol()` but we never call the `getSymbols` method.

Reply:
Went ahead and removed it completely in a [separate PR](https://github.com/prisma/prisma/pull/14853)